src/main/java/org/example/App.java

```diff
@@ -0,0 +1,90 @@
+package org.example;
+
+import java.util.*;
+
+/**
+ * This is class App which contains the main method.
+ */
+
+public class App {
+    public static void main(String[] args) {
+        Collection<Person> myTreeSet = new TreeSet<Person>(new sortPersonByName());
```

Comment:
sortPersonByName class is missing from this pull request, also .. it should start with Uppercase letter